Content: Unfettered campaign Act 1 (#6416)
endless-sky/endless-sky: data/hai/hai news.txt

		word
			`"`

news "hai joining unfettered"
	location
		government "Hai"
	name
		word
			"Hai civilian"
			"Hai youth"
	to show
		random < 5
	message
		word
			`"`
		word
			"Apparently our brothers and sisters to the north are in need of help. I may go there someday."
			"Some very big things are going to happen in Unfettered space, and I wish to be part of them!"
			"I have lost everything... I knew I should not have taken that big loan. I will be heading for Unfettered space for a new life now."
			"The ship carrying my family was attacked by pirates. I guess I may as well be joining the Unfettered now, I have nothing to lose."
			"I come from Unfettered space. The Hai there are quite dynamic and friendly once you get to know them, but I could not stand the new living conditions."

> 			"Have you heard? The art in Unfettered space is very dynamic. I wish there would be more interactions with the Unfettered."

Author: I mean it keeps changing, and isnt dull art
`"I wish I could see it in-person"` would be another possible phrase for sure

		word
			`"`

news "hai joining unfettered"
	location
		government "Hai"
	name
		word
			"Hai civilian"
			"Hai youth"
	to show
		random < 5
	message
		word
			`"`
		word
			"Apparently our brothers and sisters to the north are in need of help. I may go there someday."
			"Some very big things are going to happen in Unfettered space, and I wish to be part of them!"
			"I have lost everything... I knew I should not have taken that big loan. I will be heading for Unfettered space for a new life now."

> 			"The ship carrying my family was attacked by pirates. I guess I may as well be joining the Unfettered now, I have nothing to lose."

Author: `random < 5`, but maybe I should weight this one even lower?

> 			"The ship carrying my family was attacked by pirates. I guess I may as well be joining the Unfettered now, I have nothing to lose."

Author: I clarified it, here is what it looks like now:
```
news "hai attacked by pirates"
	location
		government "Hai"
	to show
		random < 1
	name
		word
			"Hai civilian"
			"Hai youth"
	message
		word
			"The ship carrying my family was attacked by pirates when they came back from human space. I guess I may as well be joining the Unfettered now, I have nothing to lose."
			"Pirates destroyed the ship of relatives. My family was always curious about the humans, and look what that got them! I will join the Unfettered, they are right."
```
The ship did not get destroyed in Hai space indeed

> 			"The ship carrying my family was attacked by pirates. I guess I may as well be joining the Unfettered now, I have nothing to lose."

Author: Just to clarify, this means they were Hai merchants/scientists, hidden in a human ship, maybe I should add one more that says that human ships are frail

			"Art in Unfettered space is very different from here, we could learn a lot from it."
			"My music is not appreciated here. I guess I'll be going to Unfettered space to see how they like it. I heard their taste is more... developed."
		word
			`"`

news "hai joining unfettered"
	location
		government "Hai (Unfettered)"
	name
		word
			"Hai merchant"
	to show
		has "Unfettered Jump Drive 1: done"
		random < 30
	message
		word
			`"`
		word
			"So many people need relocation. I have made it my job to carry them around, asking for little if anything. Some are Hai going into Unfettered space because they have lost themselves."
			"With the rumors of action soon in the north, the demand in transport here has greatly increased. I cannot possibly fit all of them. Time to buy a new ship, I guess."
			"Many of those who transport Hai here ask for little - too little. Have you seen the increase in demand? I'm going to raise my price; I can't be transporting people around all day for nothing."
			"Once I'm done transporting people, I will stay here and be a part of the action."

> 			"I've just finished transporting Hai, and now I have to transport others? Maybe you could help us, captain."

Reviewer: ```suggestion
			"I've just finished transporting some passengers to Unfettered space, and now I have to transport more? Maybe you could help us, Captain."
```

		word
			`"`

news "hai attacked by pirates"
	location
		government "Hai"
	to show
		random < 1
	name
		word
			"Hai civilian"
			"Hai youth"
	message
		word
			`"`
		word

> 			"The human ship carrying my family was attacked by pirates when they came back from human space. I guess I may as well be joining the Unfettered now, I have nothing to lose."

Reviewer 2: While the comma splice is technically incorrect, this is spoken dialogue and paced to match. I think this works better as is.

> 			"The human ship carrying my family was attacked by pirates when they came back from human space. I guess I may as well be joining the Unfettered now, I have nothing to lose."

Reviewer 3: I disagree, but here's an alternate suggestion. In either case, `might as well be joining` is awkward wording and should be changed.
```suggestion
			"The human ship carrying my family was attacked by pirates when they came back from human space. I guess I may as well be join the Unfettered now that I have nothing to lose."
```

> 			"The human ship carrying my family was attacked by pirates when they came back from human space. I guess I may as well be joining the Unfettered now, I have nothing to lose."

Reviewer 2: I am of the opinion that Hai speaking English should occasionally be awkward (especially if not any more technically incorrect than other spoken dialogue in general) and not always in consistent ways. :)

> 			"Pirates destroyed the human ship my children were on. They were always curious about the humans, and look what that got them! I will join the Unfettered; they are right."

Reviewer 4: ```suggestion
			"Pirates destroyed the human ship my children were on. They were always curious about the humans, and look where that got them! I will join the Unfettered; they are right."
```

			"Human ships are so frail... My sister was on one, and she told me it lost all of its shields in a few shots! Thankfully, human weapons are not that strong either, so they were able to escape."
			"You call these human vessels ships? They just barely protect against the void. Given the danger of human space, I would expect humans to have more heavy transports that can at least try to defend themselves, such as the Behemoth!"
			"No wonder you have so many pirates in your space, with those fragile freighters you have. So much cargo with so little protection is pure madness! That is why I love humans."
		word
			`"`

news "hai joining unfettered"
	location
		government "Hai"
	name
		word
			"Hai civilian"
			"Hai youth"
	to show
		random < 5
	message
		word
			`"`
		word
			"Apparently our brothers and sisters to the north are in need of help. I may go there someday."
			"Some very big things are going to happen in Unfettered space, and I wish to be a part of it!"
			"I have lost everything... I knew I should not have taken that big loan. I will be heading to Unfettered space for a new life now."

> 			"I come from Unfettered space. The Hai there are quite dynamic and friendly once you get to know them, but I could not stand the poor living conditions."

Author: that could work but what I meant is they're... active, I guess? energetic?

> 			"I come from Unfettered space. The Hai there are quite dynamic and friendly once you get to know them, but I could not stand the poor living conditions."

Reviewer 2: I think this works just fine as is, even though it is indeed an uncommon way to phrase it.

"dynamic  
/daɪˈnæmɪk/
/daɪˈnæmɪk/
[IPA guide](https://www.vocabulary.com/resources/ipa-pronunciation/)
Other forms: dynamics

If a person, place, or thing is energetic and active, then it's dynamic. When things are dynamic, there's a lot going on.

Even the most exciting rural town won't be as dynamic as a big city: there are just more things happening in a city. Someone with a dynamic personality is probably funny, loud, and excitable; a quiet, mousy person isn't dynamic. You can also talk about the dynamic aspect of music, which has to do with how the music uses dynamics, which means "changes in volume." When things are dynamic, changes and energy are in the air."

Outgoing may be a surface level synonym, but it is really not appropriate tonally here as a swap.

If anything, "energetic" would be my pick, though "energzied" or "driven" would also work and give slightly different implications.

		word
			`"`

news "hai attacked by pirates"
	location
		government "Hai"
	to show
		random < 1
	name
		word
			"Hai civilian"
			"Hai youth"
	message
		word
			`"`
		word
			"The human ship carrying my family was attacked by pirates when they came back from human space. I guess I may as well be joining the Unfettered now, I have nothing to lose."
			"Pirates destroyed the human ship my children were on. They were always curious about the humans, and look what that got them! I will join the Unfettered; they are right."

> 			"Human ships are so frail... My sister was on one, and she told me it lost all of his shields in a few shots! Thankfully, human weapons are not that strong either, so they were able to escape."

Author: You mean say it was the pilot's skill that allowed her escape, and that the sister is better than the one speaking to you?

> 			"Human ships are so frail... My sister was on one, and she told me it lost all of his shields in a few shots! Thankfully, human weapons are not that strong either, so they were able to escape."

Reviewer 4: In my interpretation, your sentence means `the ship lost all of the ship's shields`. I have no idea who `he` could be in your sentence, since ships (and spaceships) are generally treated as female in English. So it is either `it lost all of its shields` (as an object) or `she lost all of her shields`.

The second sentence is fine. When I said `she/her would be even better`, I was referring to the pronouns of the spaceship.

			"My music is not appreciated here. I guess I'll be going to Unfettered space to see how they like it. I heard their taste is more... developed."
		word
			`"`

news "hai joining unfettered"
	location
		government "Hai (Unfettered)"
	name
		word
			"Hai merchant"
	to show
		has "Unfettered Jump Drive 1: done"
		random < 30
	message
		word
			`"`
		word
			"So many people need relocation. I have made it my job to carry them around, asking for little if anything. Some are Hai going into Unfettered space because they have lost themselves."
			"With the rumors of action soon here in the north, the demand for transportation has greatly increased. I cannot possibly fit all of them. Time to buy a new ship, I guess."
			"Many of those who transport Hai here ask for little - too little. Have you seen the increase in demand? I'm going to raise my price; I can't be transporting people around all day for nothing."
			"Once I'm done transporting people, I will stay here and be a part of the action."
			"I've just finished transporting some passengers to Unfettered space, and now I have to transport more? Maybe you could help us, Captain."

> 			"Is it just me, or are there fewer merchants transporting Hai here?"

Author: Less many?

> 			"Is it just me, or are there fewer merchants transporting Hai here?"

Reviewer 4: I mean, isn't it supposed to be more?

Based on your reaction that isn't the case though, so nevermind.

> 			"Is it just me, or are there fewer merchants transporting Hai here?"

Author: Well the fact that its decreasing explains why they would need a lowly human to do it
Some of the supporters of the UHai are tired of this permanent conflict, and prefer to do more lucrative, less risky things, such as bringing tourists anywhere, instead of bringing those UHai sometimes for free even
